src/main/webapp/index.html

```diff
@@ -2,7 +2,8 @@
   <head>
     <meta charset="UTF-8">
     <title>Home</title>
+    <link rel="stylesheet" href="style.css">
   </head>
-    <h1 class ="name">Hello World</h1>    
+    <h1 class ="name">Events</h1>    
   </body>
 </html>
```

Comment:
Maybe we need to have a naming convention on the files, like:
step1.html for step 1
step2.html for step 2
step3.html for step 3